test/libsolidity/semanticTests/getters/struct_with_bytes_simple.sol

```diff
@@ -0,0 +1,17 @@
+contract C {
+    struct S {
+        uint a;
+        bytes b;
+        mapping(uint => uint) c;
+        uint[] d;
+    }
+    uint shifter;
+    S public s;
+    constructor() public {
+        s.a = 7;
+        s.b = "abc";
+        s.c[0] = 9;
+    }
+}
+// ----
+// s() -> 0x07, 0x40, 0x03, 0x6162630000000000000000000000000000000000000000000000000000000000
```

Comment:
Ok, it may make sense to drop ``S.c``, but what happens to ``S.d``?

Reply:
I mean it seems like we have always done this and also for even ABIEncoderV2, but why? It's weird...

Reply:
Getters generate code that corresponds to `x[a][b][c][d][e]` and since `S.d` is a dynamically-sized array, it can only be returned in full which is not done for arrays except for `bytes` and `string` where you would not expect to get individual elements anyway.